config/k8s/config.yaml

```diff
@@ -7,3 +7,4 @@ serviceProviders:
   clientId: "456"
   clientSecret: "54"
 baseUrl: https://${OAUTH_HOST}
+vaultHost: http://${VAULT_HOST}
```

Comment:
we can, but it would require generating all the certificates, mounting certificate authority to operator and oauth etc. As this is only for local development, I'm not sure it's worth to do it now. On real cluster with real certs, this will run on https (https://github.com/redhat-appstudio/infra-deployments/pull/489)

Reply:
I've fixed it by passing `--dev-mode` to the container, which sets vault client to ignore untrusted cert. In your PR you've set dedicated parameter to that https://github.com/redhat-appstudio/service-provider-integration-operator/pull/172/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261R65

I'm not sure which one is better.